leshan-server-cluster/src/main/resources/log4j2.xml

```diff
@@ -32,6 +32,9 @@ Contributors:
     <logger name="org.eclipse.leshan.server.security.SecurityCheck" level="trace" additivity="false">
         <AppenderRef ref="STDOUT"/>
     </logger>
+    <logger name="org.eclipse.leshan.core.model.LwM2mModel" level="trace" additivity="false">
```

Comment:
Actually, I would like to report to demos app users  when they override an embedded model.

I want this [log](https://github.com/eclipse/leshan/blob/master/leshan-core/src/main/java/org/eclipse/leshan/core/model/LwM2mModel.java#L48). This is `debug` not `trace`. As `trace` include `debug`, it works. But I can change it to avoid confusion.